modules/metrics/src/main/java/com/spotify/apollo/metrics/semantic/MetricsConfig.java

```diff
@@ -87,4 +95,8 @@
   public Set<Integer> precreateCodes() {
     return precreateCodes;
   }
+
+  public int reservoirTtl() {
+    return reservoirTtl;
```

Comment:
We have a helper function in [ConfigUtil](https://github.com/spotify/apollo/blob/1.x/apollo-environment/src/main/java/com/spotify/apollo/environment/ConfigUtil.java#L40) that you can use instead of the conditional above.
 
```suggestion
    return optionalInt("metrics.reservoir-ttl").orElse(DEFAULT_TTL_SECONDS);
```

Reply:
fixed 👍